satpy/readers/nwcsaf_hrw_nc.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2025- Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""NWC SAF GEO v2021 HRW.
+
+This module contains the :class:`NWCSAFGEOHRWFileHandler` file handler for the High Resolution Winds (HRW)
+data produced with NWC SAF GEO.
+
+There are 37 different quantities in the files, and predicted trajectories for each
+observation. Reading of the trajectory data are not currently supported.
+
+The data in the files are grouped separately for each original imaging channel, which there are up to
+seven. The number of channels depends on the configuration of NWC SAF GEO software.
+
+By default the reader treats each imaging channel separately, and thus lists
+7 (channels) * 37 (variables) = 259 distinct datasets::
+
+    import pprint
+    from satpy import Scene
+
+    filename = "S_NWC_HRW_MSG3_MSG-N-BS_20250206T130000Z.nc"
+    scn = Scene(reader="nwcsaf-geo", filenames=[filename])
+    pprint.pprint(scn.available_dataset_names())
+
+This print all the available datasets. The truncated output of this is::
```

Comment:
```suggestion
This prints all the available datasets. The truncated output of this is::
```

Reply:
Fixed in https://github.com/pytroll/satpy/pull/3070/commits/6e690d96d37ba79899b9bc6c73abfe1867fee1f2